sdk/eventhub/Microsoft.Azure.WebJobs.Extensions.EventHubs/CHANGELOG.md

```diff
@@ -4,10 +4,14 @@
 
 ### Features Added
 
+- Added the ability to bind to the `EventHubAsyncCollector` for output, which supports an overload allowing a partition key to be specified.
```

Comment:
Note to self:  Reword this if we decide to move forward with the extension method.